display.php

```diff
@@ -0,0 +1,37 @@
+<?php 
+require_once 'functions.php';
+
+$db = new PDO('mysql:host=db; dbname=cuttlefish', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `name`, `points`, `games`,`rings` FROM `nba`");
+$query->execute();
+$players = $query->fetchAll();
```

Comment:
And then I would suggest moving these lines into a new function as well, so you can easily get the list of data

Reply:
sorted also